README.md

```diff
@@ -384,6 +384,28 @@ that you want to force the cache to rebuild.  You can do this by using the
 tables from the cache, re-create the tables, then populate the cache with the
 data that is in the politeiad git repositories.
 
+##### Building with repository version
+
+It is often useful to have version information from the repository where 
+politeia was fetched and built from, such as the commit hash it is using.  
+To accomplish this, politeia needs to be built with `go get` from outside 
+of your local files. If you build using your local checked out repository, 
+the build information will return `(devel)` instead of the actual version, 
+since it was builded locally on your development environment. If built 
+properly, and suppose politeia has a release of the version 1.0.0, it will 
+return `v1.0.0-<date>-<short commit hash>`. This build version is logged on 
+startup and returned from the version API call. Below are examples on how
+to build politeia from outside of `GOPATH` and your local repository:
+
+`GO111MODULE=on go get github.com/decred/politeia/politeiad/cmd/...@master`
```

Comment:
Why are you only getting the politeiad commands here?  Shouldn't you be getting the entire politeia repo?

Reply:
This command builds the entire politeia stack. My thought is it does this because of the dependencies they have between each other.

see https://matrix.to/#/!VFRvyndKpzcLrVslQD:decred.org/$158454223040232JLAHv:decred.org?via=decred.org&via=matrix.org&via=zettaport.com

Reply:
Oh ok.  That's fine then.